docs/walkthrough/working-with-the-catalog.md

```diff
@@ -1,35 +1,44 @@
-<!-- markdownlint-disable -->
-The Quilt Catalog is the second half of Quilt. It provides an interface on top of your S3 bucket that brings Quilt features like data packages and search to the web.
+<!-- markdownlint-disable-next-line first-line-h1 -->
+The Quilt Catalog is the second half of Quilt. It provides an interface on top
+of your S3 bucket that brings Quilt features like data packages and search to
+the web.
 
 **[For a hands-on demo, check out the public demo catalog](https://open.quiltdata.com/b/quilt-example).**
 
-Note that you can use the Quilt Python API without using the catalog product, but they are designed to work together.
+Note that you can use the Quilt Python API without using the catalog product,
+but they are designed to work together.
 
 ## Brief tour
 
-The Quilt catalog provides a homepage for your catalog, based on a `README.md` file that you can optionally create at the top of your bucket.
+The Quilt catalog provides a homepage for your catalog, based on a `README.md`
+file that you can optionally create at the top of your bucket.
 
 ### Browse
 
-![](../imgs/catalog_homepage.png)
+![Homepage](../imgs/catalog_homepage.png)
```

Comment:
Needs to be replaced with a new screenshot - we don't have a "Files" tab anymore, it's been replaced with "Bucket"